src/main/java/duke/DialogBox.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.collections.FXCollections;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+import javafx.collections.ObservableList;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.image.Image;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.shape.Circle;
+import javafx.scene.shape.Rectangle;
+import java.io.IOException;
+import java.util.Collections;
+
+/**
+ * Represents a dialog box consisting of an ImageView to represent the speaker's face and a label
+ * containing text from the speaker.
+ */
+public class DialogBox extends HBox {
+    @FXML
+    private Label dialog;
+    @FXML
+    private ImageView displayPicture;
+
+    public DialogBox(String input, Image image) {
+        try {
+            FXMLLoader fxmlLoader = new FXMLLoader(MainWindow.class.getResource("/view/DialogBox.fxml"));
+            fxmlLoader.setController(this);
+            fxmlLoader.setRoot(this);
+            fxmlLoader.load();
+            dialog.setText(input);
+            Circle clip = new Circle(displayPicture.getFitWidth()/2,
+                    displayPicture.getFitHeight()/2, displayPicture.getFitWidth()/2);
```

Comment:
I noticed that the '/' was not preceded and followed by whitespace in this class. Great job on ensuring that operators are surrounded by whitespace in other classes 👍

Reply:
Thank you for pointing it out! I have changed it to follow the coding standard.